include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -341,8 +341,7 @@ struct tuple<T1, T...>
     template <typename _U1, typename... _U,
               typename = ::std::enable_if_t<
                   (sizeof...(_U) == sizeof...(T) &&
-                   oneapi::dpl::__internal::__conjunction<::std::is_constructible<T1, _U1&&>,
-                                                          ::std::is_constructible<T, _U&&>...>::value)>>
+                   std::conjunction_v<::std::is_constructible<T1, _U1&&>, ::std::is_constructible<T, _U&&>...>)>>
```

Comment:
Since we only use conjunction value here, not an integral constant itself, may be it makes sense to simply use fold expression here:

```suggestion
                   ::std::is_constructible_v<T1, _U1&&> &&
                   ( ::std::is_constructible_v<T, _U&&> && ... )
```

Reply:
I propose to apply this change in separate PR.
Let's save the goal of this PR simple and clear.

Reply:
Let's not use fold. `conjunction` is better in a sense of template instantiations. It stops instantiating the tail when meets first `false`, while fold-expression instantiates all the templates always.

Reply:
@rarutyun, @kboyarinov  so looks like the current state of this PR is ok?

Reply:
Agree, let's keep everything as-is.